### DISCUSSION TITLE
Trying to revert to an earlier iteration of Visual Studio Code

### DISCUSSION THREAD

**Discussion by axtens** [ScoopInstaller/Main#6064] [Q&A] [answered]
I tried `scoop install vscode@1.88.0` but got a hash check failure with a link that created a hash check error for vscode@1.91.1

I'm running vscode on Windows Server 2012R2 and it appears that vscode no longer supports that O/S if the error I get when trying to run v1.91.1 is anything to go by, viz: `The procedure entry point DiscardVirtualMemory could not be located in the dynamic link library Code.exe`

How do I revert to an earlier version of VSC?

**Comment by Ra2-IFV** [comment]
Well, I think you should download binary manually, since scoop becomes useless for this package

**Comment by HUMORCE** [comment, marked as answer]
`-s, --skip-hash-check           Skip hash validation (use with caution!)`

**Comment by Lutra-Fs** [comment]
Hopefully you will also benefit from https://github.com/ScoopInstaller/Scoop/pull/6370#issue-comment-box

**Comment by z-Fng** [comment]
> I tried `scoop install vscode@1.88.0` but got a hash check failure with a link that created a hash check error for vscode@1.91.1

- https://github.com/ScoopInstaller/Extras/pull/16921
